producer-c/producer-cloudwatch-integ/CanaryStreamCallbacks.h

```diff
@@ -0,0 +1,72 @@
+#ifndef __KINESIS_VIDEO_CONTINUOUS_RETRY_INCLUDE_I__
+#define __KINESIS_VIDEO_CONTINUOUS_RETRY_INCLUDE_I__
+
+#pragma once
+
+#include <com/amazonaws/kinesis/video/cproducer/Include.h>
```

Comment:
have a single include file that pulls in all of the include files. All of the sources then will include that include. Take a look at the codebase. Most of the source files include a single Include_i.h or similar. That one then pulls in all of the other headers.